app-i18n/man-pages-it/Manifest

```diff
@@ -1,2 +1,2 @@
 DIST man-pages-it-2.80.tar.gz 1789480 BLAKE2B 664121e6a6d8e73200546431ea97d0b8c751eaeed87e958ff0b0a822c651c7e489b05a7e26de23a889a4f1ab6fc21524bb81fab21f0331eedb4102d40cecfeb1 SHA512 81cee1b4d0c818b83830355ba1347df7a309b87ebfe3169db7763b22de8638273eacf377d0b9127aa1e748c8fd427eb268dda6d6c4b543c6d3788edda86980ef
-DIST man-pages-it-4.00.tar.xz 349192 BLAKE2B 90f25e7abfd64d0f175c2596bc9203626d9e7e53218d579081d17aa01665be20ffa721e33379107814a57715a35f480bb1833a2d3965f4b86b3247bf70e9162c SHA512 6392b42254eba42bd76f3aef3b6206396a817019df61462e44bd0f7dc519768cb1b1107726c45e402a0cf27e6f2ca8c8912363bd9bec7562b10e8fd5dcc60ffe
```

Comment:
Please split version bumps into separate commits adding new versions and removing old versions. Also, preferably please don't remove the old version immediately but instead give it a few days to make sure users can downgrade if the new version causes trouble.